mass_editing/tests/test_mass_editing.py

```diff
@@ -38,33 +53,52 @@ def _create_partner(self):
             'email': 'example@yourcompany.com',
             'phone': 123456,
             'category_id': [(6, 0, categ_ids)],
+            'notify_email': 'always'
+        })
+
+    def _create_partner_title(self):
+        """Create a Partner Title."""
+
+        # Loads German to work with translations
+        self.lang_model.load_lang('de_DE')
+
+        partner_title = self.res_partner_title_model.create({
+            'name': 'Ambassador',
+            'shortcut': 'Amb.',
         })
 
+        # Adding translated terms
+        ctx = {'lang': 'de_DE'}
+        partner_title.with_context(ctx).write({
+            'name': 'Botschafter',
+            'shortcut': 'Bots.'})
+        return partner_title
+
     def _create_user(self):
         return self.env['res.users'].create({
             'name': 'Test User',
             'login': 'test_login',
             'email': 'test@test.com',
         })
 
-    def _create_mass_editing(self, model, fields):
+    def _create_mass_editing(self, model, fields, model_name):
         """Create a Mass Editing with Partner as model and
         email field of partner."""
         mass = self.mass_object_model.create({
-            'name': 'Mass Editing for Partner',
+            'name': u'Mass Editing for {0}'.format(model_name),
```

Comment:
In Python 3, there's no more `u''`, as everything is unicode.

Reply:
Ok, thanks